BonusCalcApi/serverless.yml

```diff
@@ -28,8 +28,8 @@ functions:
     role: lambdaExecutionRole
     environment:
 # TODO: Create ssm variables for this API's postgres mirror then rename bonuscalc-api below to match api name

```

Comment:
Not sure why git has flagged this one as a change, the two texts are identical. Perhaps a forgotten `dotnet format` from another PR